web/pdf_find_controller.js

```diff
@@ -517,6 +517,14 @@ class PDFFindController {
     });
   }
 
+  /**
+   * Updates scroll so that the match appears into view.
+   *
+   * @param {HTMLElement} element
+   * @param {number} selectedLeft
+   * @param {number} pageIndex
+   * @param {number} matchIndex
+   */
```

Comment:
Given that the method takes an *object*, surely this needs to be a `typedef` in order for it to work correctly!?

Reply:
Indeed, I mixed it up with `scrollIntoView`